bench.c

```diff
@@ -83,12 +83,16 @@ static void *bench_worker(__attribute__((unused)))
 
     /* wait until all workers created */
     pthread_mutex_lock(&worker_lock);
-    while (!ready)
-        if (pthread_cond_wait(&worker_wait, &worker_lock)) {
-            puts("pthread_cond_wait failed");
-            exit(-1);
-        }
+    if (++n_retry == MAX_THREAD)
```

Comment:
Explicitly add curly brackets (i.e., `{` and `}`) at the end of `if`, `else`, and `while` statements to prevent [dangling else](https://en.wikipedia.org/wiki/Dangling_else).